exercises/isbn-verifier/description.md

```diff
@@ -1,4 +1,7 @@
-Check if a given ISBN-10 is valid.
+Given a number, check if a given ISBN-10 is valid.
```

Comment:
I'm not sure this change is an improvement.

Reply:
Actually the whole line can probably be removed.

Reply:
Sure thing, I was just trying to mirror the [Luhn description](https://github.com/exercism/problem-specifications/blob/master/exercises/luhn/description.md), as I find it a really good example for this kind of problem.